proto/tendermint/privval/types.proto

```diff
@@ -5,15 +5,6 @@ import "tendermint/crypto/keys.proto";
 import "tendermint/types/types.proto";
 import "gogoproto/gogo.proto";
 
-enum Errors {
```

Comment:
unused?

Reply:
Yes, I believe this enum is not used in Go code as well, at least not in code using `cometbft.privval.v1`. As `tendermint.privval` should be in sync with v1, it was left there by oversight.

Reply:
Shouldn't we nuke all tendermint protos at this stage?